package bespalhuk.kcoroutines.core.common.exception

class UserAlreadyExistsException : RuntimeException("User already exists.")
